components/layout/flow/mod.rs

```diff
@@ -308,7 +308,7 @@ pub(crate) struct CollapsibleWithParentStartMargin(bool);
 pub(crate) struct OutsideMarker {
     pub list_item_style: Arc<ComputedValues>,
     pub base: LayoutBoxBase,
-    pub block_container: BlockContainer,
+    pub bfc: BlockFormattingContext,
```

Comment:
Nit: Let's avoid the abbreviation for clarity:

```suggestion
    pub block_formatting_context: BlockFormattingContext,
```